<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>automation_widget</class>
 <widget class="QWidget" name="automation_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="automation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="3">
       <widget class="QPushButton" name="load_experiment_button">
        <property name="text">
         <string>Load Experiment</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2" colspan="4">
       <widget class="QTabWidget" name="auto_protocol_tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="DLP_configuration">
         <attribute name="title">
          <string>DLP_configuration</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="2" column="5">
           <widget class="QLineEdit" name="number_sequence_repetition_lineEdit"/>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="dlp_mode_comboBox">
            <item>
             <property name="text">
              <string>Static image</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HDMI video</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pattern Sequence</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="intervalle_between_sequences_label">
            <property name="text">
             <string>Intervalle between sequences</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="dlp_mode_label">
            <property name="text">
             <string>DLP mode</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="number_of_sequence_repetition_label">
            <property name="text">
             <string>Number of  sequence repetition</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="QLineEdit" name="experiment_repetition_lineEdit"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="number_of_sequence_lineEdit"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="dlp_time_on_lineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="dlp_time_off_label">
            <property name="text">
             <string>DLP time OFF</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="dlp_time_on_label">
            <property name="text">
             <string>DLP time ON</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="number_of_sequence_label">
            <property name="text">
             <string>Number of sequence</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="dlp_time_off_lineEdit"/>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="experiment_repetition_label">
            <property name="text">
             <string>Experiment repetitions</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLineEdit" name="intervalle_between_sequences_lineEdit"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Scanning">
         <attribute name="title">
          <string>Scanning</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="1">
           <widget class="QPushButton" name="stop_scan_pushButton">
            <property name="text">
             <string>Stop Scan</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2" alignment="Qt::AlignVCenter">
           <widget class="QProgressBar" name="scan_progress_progressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="start_scan_pushButton">
            <property name="text">
             <string>Start Scan</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="export_experiment_button">
        <property name="text">
         <string>Export Experiment</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="run_experiment_button">
        <property name="text">
         <string>Run Experiment</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="automatic_protocols_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Automatic protocols</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
